.travis.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       script:
       - pip3 install anybadge==1.1.1 pylint==2.3.0 pylint-exit==1.0.0
       - pip3 freeze
-      - pylint --rcfile=.pylintrc --output-format=text src | tee pylint.txt || pylint-exit $?
+      - pylint --rcfile=.pylintrc --output-format=text src | tee pylint.txt
```

Comment:
Just a question what is the reason we removed this ?

Reply:
the command is supposed to exit with a code greater than zero if any issue comes up, not only if errors occur